.github/workflows/build.yml

```diff
@@ -21,33 +37,42 @@ jobs:
           - "3.9"
           - "3.10"
           - "3.11"
+          - "3.12"
     steps:
-      - uses: satackey/action-docker-layer-caching@v0.0.11
-        continue-on-error: true
-        with:
-          key: thumbor-docker-${{ matrix.image-tag }}-{hash}
-          restore-keys: |
-            thumbor-docker-${{ matrix.image-tag }}-
+      # - uses: satackey/action-docker-layer-caching@v0.0.11
+      #   continue-on-error: true
+      #   with:
+      #     key: thumbor-docker-${{ matrix.image-tag }}-{hash}
+      #     restore-keys: |
+      #       thumbor-docker-${{ matrix.image-tag }}-
       - uses: actions/checkout@v4
-      - name: Build test image
-        run: docker build -t test_image -f TestDockerfile --build-arg PYTHON_VERSION=${{ matrix.image-tag }} .
-      - name: Start test container
-        run: |
-          docker run -dt --name thumbor_test -v $(pwd):/app test_image
-      - name: Fire up Redis
-        run: docker exec thumbor_test make redis
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.image-tag }}
+      - name: Install SO dependencies
+        run: sudo apt-get update && sudo apt-get install -y libwebp-dev libjpeg-dev libssl-dev zlib1g-dev ffmpeg scons gifsicle libcurl4-openssl-dev libgnutls28-dev libjpeg-progs libimage-exiftool-perl
+      # - name: Build test image
+      #   run: docker build -t test_image -f TestDockerfile --build-arg PYTHON_VERSION=${{ matrix.image-tag }} .
+      # - name: Start test container
+      #   run: |
+      #     docker run -dt --name thumbor_test -v $(pwd):/app test_image
+      # - name: Fire up Redis
+      #   # run: docker exec thumbor_test make redis
```

Comment:
Hey @jairhenrique,

Thumbor needs redis single node and sentinel instance running to test itself.

Maybe another service with sentinel configs should work.

Reply:
I remove this commit, it's only a test.